src/test/java/org/sdo/rendezvous/repositories/JedisRepositoryTest.java

```diff
@@ -43,7 +43,8 @@
   private static String MOCKED_RANDOM_SEQUENCE = "aaaaaaaaaa";
   private static String MOCKED_LOG_RECORD =
       "{\"guid\":\"00000000-0000-0000-0000-000000000003\","
-          + "\"ct\":\"Tue May 09 10:33:39 CEST 2017\",\"gid\":\"00000000000000000000000000001234\"}";
+          + "\"ct\":\"Tue May 09 10:33:39 CEST 2017\","
+             + "\"gid\":\"00000000000000000000000000001234\"}";
```

Comment:
Improper indent, both lines 46 and 47 should have same level of indent.

Reply:
Corrected